<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Disposable</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/MainImage/256x256.png</normaloff>:/MainImage/256x256.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="identitiestab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="maintab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="res.qrc">:/MainImage/MainImage.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIdentity">
    <property name="title">
     <string>&amp;Identity</string>
    </property>
    <addaction name="newidentity_btn"/>
    <addaction name="separator"/>
    <addaction name="showhash_btn"/>
    <addaction name="renameidentity_btn"/>
    <addaction name="deleteidentity_btn"/>
   </widget>
   <addaction name="menuIdentity"/>
  </widget>
  <action name="newidentity_btn">
   <property name="text">
    <string>New identity</string>
   </property>
  </action>
  <action name="deleteidentity_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="renameidentity_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="newchat">
   <property name="text">
    <string>New chat</string>
   </property>
  </action>
  <action name="showhash_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show hash</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
